react-app/src/reportWebVitals.js

```diff
@@ -0,0 +1,13 @@
+const reportWebVitals = onPerfEntry => {
+  if (onPerfEntry && onPerfEntry instanceof Function) {
+    import('web-vitals').then(({ getCLS, getFID, getFCP, getLCP, getTTFB }) => {
+      getCLS(onPerfEntry);
+      getFID(onPerfEntry);
+      getFCP(onPerfEntry);
+      getLCP(onPerfEntry);
+      getTTFB(onPerfEntry);
+    });
+  }
+};
+
+export default reportWebVitals;
```

Comment:
Web Vitals are some metrics to measure the performance of your app, and you can either `conolse.log` it in your `index.js` file or just comment it out there if you don't want to see any performance details. More on that here:
https://create-react-app.dev/docs/measuring-performance/

Reply:
done!